### PR TITLE
Create README.md

### LINKED ISSUE #800: Community Call 30 March
Our next 1h community call to chat about recent developments in Frictionless and data generally and next steps, involving our community members in the process!

Organizer: @sapetti9 (OKF) 

Date: **30<sup>th</sup> March, 4 pm CET**

## Agenda 

Join us to hear about Frictionless’ new features: the metadata editor, and the new CLI commands. We will also discuss the project governance during the call.

RSVP to get the meeting link here: https://forms.gle/k7AsKKD2tu6rm8TR6

Social Media [@Frictionlessd8a](https://twitter.com/Frictionlessd8a), [Slack Community chat](https://frictionlessdata.slack.com/messages/general), [Matrix bridge](https://matrix.to/#/#frictionlessdata:matrix.okfn.org)

## Tasks

* [x] Zoom call link. – 
  * [x] Reserve slot in Zoom account. – 
  * [x] Email guests ~2-3h before the event with the meeting details. – 
* [x] Announce on Twitter. – 
* [x] Send reminders on Slack.
  * [x] Send reminder to register a few days / a week before the call. – 
  * [x] Send meeting link ~10 minutes before the event on Slack. – 
* [x] Recording on YouTube. – 
  * [x] Process video(s) (cut presentation out as a separate video if any, trim beginning and end if necessary). –
  * [x] Update blog post with link to YouTube video. – 
* [x] Update RSVP form with the correct date. – 
* [x] Add previous guests to the Google Calendar invite (make sure Google Meet is turned off!). – 
  * [x] Notify guests by email about the calendar event –
* [x] Create a new entry in the public meeting doc with agenda for this call. –

### PR DESCRIPTION
**Pull request by sapetti9**
Adding a blog for the March community call. @aivuk can you have a look and merge if OK? Thanks

### CODE REVIEW

File: site/blog/2023-04-06-march-community-call/README.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
+---
+title: Frictionless Data Community Call March 2023
+date: 2023-04-06
+tags: ['events', 'community-hangout']
+category: events
+image: /img/blog/Frictionless-new-CLI-commands.png
+description: At our last community call Evgeny Karev presented the new Frictionless CLI commands and we talked with the community about project governance...
+author: Sara Petti
+---
+At our last community call on March 30<sup>th</sup>, our very own Evgeny Karev - tech lead of the Frictionless Data project at [Open Knowledge Foundation](http://okfn.org/), presented the new Frictionless command line features.
```
Comment (aivuk) [inline site/blog/2023-04-06-march-community-call/README.md:10]: This sentence is confusing to me: our very own Evgeny? I don't know exactly what you had in mind to write. Would it be something like: Evegny, our own tech lead of the frictionless project?

Comment (aivuk) [reply site/blog/2023-04-06-march-community-call/README.md:10]: I think just, the tech lead of the frictionless project would be enough

Comment (sapetti9) [reply site/blog/2023-04-06-march-community-call/README.md:10]: It's an English idiom. It's basically saying that Evgeny and the writer (me) are in the same group (=Frictionless Data/OKFN). It's pretty common in informal English, and I like to give to the blog some informal twists, but I can delete it if you think it's confusing.

Comment (aivuk) [reply site/blog/2023-04-06-march-community-call/README.md:10]: Got it, thanks. I don't see any problem then! :+1:

**Review by aivuk — APPROVED** [review]
Looks good!

**Pull request merged**